tests/_test_mscolab/test_server.py

```diff
@@ -410,5 +443,5 @@ def _save_content(self, operation, userdata=None):
             userdata = self.userdata
         user = get_user(userdata[0])
         fm = FileManager(self.app.config["MSCOLAB_DATA_DIR"])
```

Comment:
The `SocketsManager`, `FileManager` and `ChatManager` instances are only really designed to be singleton objects at the moment, creating more instances of them does not reflect reality, so isn't useful for tests, and might even break stuff as it did with multiple `SocketsManager`s.